pkg/pdutil/pd.go

```diff
@@ -86,8 +95,8 @@ var (
 // pdHTTPRequest defines the interface to send a request to pd and return the result in bytes.
 type pdHTTPRequest func(context.Context, string, string, *http.Client, string, io.Reader) ([]byte, error)
 
-// pdRequest is a func to send a HTTP to pd and return the result bytes.
-func pdRequest(
+// PDRequest is a func to send a HTTP to pd and return the result bytes.
+func PDRequest(
```

Comment:
We needn't expose this function as public, see below

Reply:
Fixed